data/core/units/dwarves/Miner.cfg

```diff
@@ -6,15 +6,19 @@
     image=units/dwarves/miner.png
     hitpoints=25
     movement_type=dwarvishfoot
+    [resistance]
+        blade=90
+        pierce=90
+        impact=90
+    [/resistance]
     movement=4
-    level=1
+    level=0
     alignment=neutral
-    advances_to=null
-    {AMLA_DEFAULT}
-    experience=50
-    cost=5
+    advances_to=Dwarvish Fighter,Dwarvish Guardsman,Dwarvish Ulfserker
```

Comment:
Why not Dwarvish Scout too?

Reply:
I’ve replaced the Dwarvish Ulfserker with a Dwarvish Scout, which makes more sense to me personally.